examples/gallery/viz/GraphViz.ipynb

```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc69e05f-be77-4c75-9157-60c60dcba6a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graphviz import Graph\n",
```

Comment:
This requires `python-graphviz` but you are installing `pygraphviz`, they are distinct libraries.

Reply:
I must admit I am confused by the graphviz/ pygraphviz/ pygraphviz and pip vs conda story. 

But at least it works on Binder **without** conda installing `python-graphviz` and by pip installing `graphviz`+`pygraphviz`. So I would claim `python-graphviz` is not absolute required ??? The problem with `python-graphviz` is that its a conda only thing and wont be available for use in the `setup.py` file.

I am simple in doubt on what to do here because of all that confusion. Can you help @philippjfr?